examples/client/client/tasks.nim

```diff
@@ -92,6 +93,17 @@ proc statusContext*(arg: ContextArg) {.async, gcsafe, nimcall,
   # re/set threadvars that don't persist across waku dis/connect
   resetContext()
 
+proc updatePrices() {.async.} =
+  while statusState == StatusState.loggedin:
+    await sleepAsync(5000)
```

Comment:
I wonder if 5 seconds is too frequent. Maybe it should be 60 seconds?

Maybe the `sleepAsync` should be moved to the bottom of the loop so that as soon as `status.login` succeeds, the first update will run right away.

I don't think it's too important now, but in the future we could add a command that allows the frequency to be set by the user. The corresponding task in `client/tasks.nim` would simply update the value of a threadvar referenced by the `updatePrices` looper proc.